test/test_blocks.py

```diff
@@ -502,7 +502,11 @@ def test_slider_random_value_config(self):
     def test_io_components_attach_load_events_when_value_is_fn(self, io_components):
         interface = gr.Interface(
             lambda *args: None,
-            inputs=[comp(value=lambda: None, every=1) for comp in io_components],
```

Comment:
Just wondering why do we have to exclude `gr.Dialogue` from this list?

Reply:
And if we need to exlcude, should we just remove it from the `io_components` fixture? Seems more maintainable than adjusting the count by one manually